rocAL_pybind/examples/rocAL_api_audio_unittest.py

```diff
@@ -84,6 +87,18 @@ def audio_decoder_pipeline(path, file_list):
         num_shards=1,
         stick_to_shard=False)
 
+@pipeline_def(seed=seed)
+def downmix_pipeline(path, file_list):
```

Comment:
Instead of adding a different pipeline, we can use the audio_decoder_pipeline itself and add only an additional argument I think that should work